docs/DeveloperGuide.adoc

```diff
@@ -294,72 +296,225 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 
 [width="59%",cols="22%,<23%,<25%,<30%",options="header",]
 |=======================================================================
-|Priority |As a ... |I want to ... |So that I can...
-|`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
+|Priority |As a ... |I can ... |So that I ...
+
+|`* * *` |student |add classmate or teammate’s name and contact info |contact them easily
+
+|`* * *` |student |edit classmate or teammate’s name and contact info |Edit them if the information are changed
+
+|`* * *` |student |delete classmate or teammate’s contact |delete if not necessary anymore
+
+|`* * *` |student |categorise contacts into teammate or classmate |
+
+|`* * *` |student |add profile picture to added contact |know at a glance who is the person in my contact
+
+|`* * *` |student |edit profile picture to added contact |edit the picture if changes are necessary
+
+|`* * *` |student |delete profile picture to added contact|
+
+|`* * *` |student |add category specific description |filter out a contact’s remark according to type of contact
```

Comment:
"Category specific" - Should clarify which objects' categories that are mentioned